### PR TITLE
Removing SampledSignals.jl as Dep && Borrowing Code 

### PR DESCRIPTION
**Pull request by ashwanirathee**
Initial work on removing SampledSignals.jl dependency and code is replaced/put directly here from original SampledSignals.jl

### CODE REVIEW

**Comment by ashwanirathee** [comment]
Good thing that tests fail properly now🤣, we are close to rising from the dark age in this package

File: src/util.jl [hunk 1]
```diff
@@ -66,3 +66,95 @@ function zero_crossings(array::AbstractVector{T}, size::Int = length(array), off
     end
     result
 end
+
+# units from SampledSignals
+
+const frames = Hz*s
+const FrameQuant = DimensionlessQuantity
```
Comment (Datseris) [inline src/util.jl:73]: SInce Unitful.jl is no longer a dependency, where do these come from?

Comment (ashwanirathee) [reply src/util.jl:73]: Unit full is still there 😅

**Comment by ashwanirathee** [comment]
@jparcill  can you confirm if speedup, pitch shift, slowdown worked after your updates?

Since task of current pr is removing sampled signals dependency, I am not sure if I should update speedup and others rn as they hold lot of old, redundant pirated types (means there are functions for stft here but they basically call DSP.Peridograms.stft).

**Comment by jparcill** [comment]
Yup they work!  Not sure how to write proper tests for them but I tested out mp3's and confirmed it by ear

**Comment by Datseris** [comment]
I say let's merge this and do further updates in different PRs. This PR successfully makes passing tests and removes the SampledSignals.

**Comment by ashwanirathee** [comment]
In one of the emails, when jared sent me the ipynb he was using for testing, they do work there. But How do we compares these methods without using duration/size all the time. Librosa does the test of speedup, pitchshift, etc with duration measure only which I am not sure if it tells much about the audio.

Yeah, It would be okay if we merge this one and go to each thing one by one in the following PRs.

**Comment by rob-luke** [comment]
Hi all,

I see that SampledSignals is no longer a dependency due to this PR. And I understand the benefits of minimising dependencies in your project.

I have just received admin access to the sampled signals repository and will try to reinvigorate it over the coming weeks and months. Probably slowly at first with getting the CIs updated etc, clearing out old PRs, then moving to maintenance mode.

I don’t expect this project to add back SampledSignals as a dependency, but I would be pleased to review any PRs you may push back upstream, and discuss any common issues these two packages may have in common to find optimal solutions together and reduce duplication of our efforts. Similarly I will keep an eye on your great package here.

**Comment by Datseris** [comment]
Actually, we would be happy to return to have SampleSignals as a dependency here and remove duplicate code. Personally, I'm much in favor of that. We dropped it because we thought it wouldn't be updated anytime soon.

Once you have SS up and running just post a comment here and we can do a simple PR that restores it and removes the code we "stole" from it.

**Comment by rob-luke** [comment]
Great! I will comment back here and ping you when everything is set. Cheers

**Comment by rob-luke** [comment]
Sorry I forgot to ping back here @Datseris @jongwook @ashwani-rathee but I think SampleSignals is up and running again, CIs have been stable since https://github.com/JuliaMusic/MusicProcessing.jl/pull/23#issuecomment-835288018 and versions are tagged etc. Please report any issues you have, cheers.

**Comment by rob-luke** [comment]
FYI my next aim is to steadily increase the test coverage for SampledSignals.

**Pull request merged**